networks/suzuka/suzuka-full-node/src/partial.rs

```diff
@@ -99,13 +98,13 @@ where
 				Ok(transaction) => {
 					debug!("Got transaction: {:?}", transaction);
 
-					let serialized_aptos_transaction = serde_json::to_vec(&transaction)?;
+					let serialized_aptos_transaction = bcs::to_bytes(&transaction)?;
 					debug!("Serialized transaction: {:?}", serialized_aptos_transaction);
 					let movement_transaction = movement_types::Transaction {
-						data : serialized_aptos_transaction,
-						sequence_number : transaction.sequence_number()
+						data: serialized_aptos_transaction,
+						sequence_number: transaction.sequence_number(),
 					};
-					let serialized_transaction = serde_json::to_vec(&movement_transaction)?;
+					let serialized_transaction = bcs::to_bytes(&movement_transaction)?;
```

Comment:
Good.
I wonder if there is a non-accidental reason why we need to apply two levels of framing here. @l-monninger is the inner transaction's encoding meant to be opaque in the wire representation of `movement_types::Transaction`?

Reply:
Opened for review

Reply:
> Good. I wonder if there is a non-accidental reason why we need to apply two levels of framing here. @l-monninger is the inner transaction's encoding meant to be opaque in the wire representation of `movement_types::Transaction`?

Yes, it's meant to be opaque as the sequencer is supposed to be as general as possible and `movement_types` was not supposed to contain specific protos. But, perhaps we should change it as the types have been mirroring one another.